app/src/main/java/com/framgia/fsalon/data/model/UserRespone.java

```diff
@@ -0,0 +1,34 @@
+package com.framgia.fsalon.data.model;
+
+import com.framgia.fsalon.data.model.Token;
```

Comment:
WARNING: [ImportFromSamePackage] No need to import a type that lives in the same package
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#ImportFromSamePackage